astroplan/plots/plots.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
+                 dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) dark_plot option
+
+    Makes an airmass vs. time plot.
+
+    If an ax object is passed in, plots an additional airmass plot on top.
+    Otherwise, creates a new ax object with an airmass plot.
+
+    When a `Time` object with a single instance in time is used (e.g., if
+    len(time) = 1), a new `Time` object is created internally.  This new object
```

Comment:
@adrn  In talking with @eteq, we decided for now that something like `Time(['2010-1-1'])` would be not be treated as a scalar.  If the user wants a time window centered on `2010-1-1`, then they must pass in `Time('2010-1-1')`.  If they pass in `Time(['2010-1-1'])`, they will get an airmass plot with only 1 data point.